automation/check-patch.e2e-kubemacpool-functests.sh

```diff
@@ -17,6 +17,7 @@ teardown() {
 main() {
     # Setup CNAO and artifacts temp directory
     source automation/check-patch.setup.sh
+    source hack/components/yaml-utils.sh
```

Comment:
The reason automation/check-patch.setup.sh is sourced at this place is because it do what the comment suggests and not acting just like a library, maybe changing `source` to `./` is a better fit.
In any case you can move this line to top of the file

Reply:
I removed it entirely (this line and the first commit) thanks to @qinqon 's  great suggestion.